open-apparel-registry/open-apparel-registry
# Update Claim a Facility claim form & profile fields

## Overview

- add job_title and linkedin_profile fields to claim a facility claim form
- enable claimants to control visibility of facility website
- split facility_name field into facility_name_english and
facility_name_non_english profile fields
- add facility_workers_count profile field
- add facility_female_workers_percentage profile field
- add facility_type field & choices for facility claim profile
- add other_facility_type field for facility claim profile
- adjust display order of claim info fields on facility details screen

Connects #617 
Connects #634 

## Demo

![Screen Shot 2019-07-08 at 10 29 30 PM](https://user-images.githubusercontent.com/4165523/60854942-eb548d80-a1cf-11e9-83af-3534cf582e4f.png)

![Screen Shot 2019-07-08 at 10 29 50 PM](https://user-images.githubusercontent.com/4165523/60854948-ee4f7e00-a1cf-11e9-9ade-6e21bea50e48.png)

## Notes

There are a bunch of migrations here but I'll squash them when the other PRs are merged in -- so I didn't bother renaming anything.

I made discrete followup issues for handling

- product type #651
- production type #652
- certifications #653
- membership bodies #654

## Testing Instructions

- get this branch, then resetdb & processfixtures
- sign in and turn on claim a facility
- try to claim a facility and verify that you see the new claim form fields, with validation
- sign in as an admin and view the claim, verifying that you see the new claim form fields displayed in the dashboard
- approve the claim, then sign back in as the user who has claimed the facility and visit the claim edit profile page
- verify that you can update and save data in the new inputs
- visit the facility details page and verify that the new fields are visible (if they aren't null) and that the order matches the order requested in #617 

## Checklist
